.nvmrc

```diff
@@ -1 +1 @@
-v22
+v23
```

Comment:
issue: I believe we should not use unstable versions of node for the headless. Is there any benefit in this case?

Reply:
Did not mean to include this so I removed this change